webserver/src/server/api/api_system_state.cpp

```diff
@@ -45,3 +47,15 @@ void handleUpdateOperationMode(AsyncWebServerRequest* request, JsonVariant& body
   serializeJson(body, *response);
   request->send(response);
 }
+
+void handleUpdateTareScales(AsyncWebServerRequest* request, JsonVariant& body) {
+  LOG_INFO("Got request to mark scales tare as pending");
+
+  AsyncResponseStream* response = request->beginResponseStream("application/json");
+  SystemState command = json::mapJsonToSystemState(body);
```

Comment:
This isn't clear. You expect the client to send the whole SystemState object or just the tarePending?

Reply:
You ain't wrong, only the tarePending should be sent ofc but i had some troubles translating that bool to systemState so kinda settled sending the whole struct for now jsut to check the working stat 🤷